huntsman/observatory.py

```diff
@@ -386,6 +386,78 @@ def take_evening_flats(self,
                 self.logger.debug('Waiting for dark-field image')
                 time.sleep(1)
 
+    def take_dark_fields(self,
+                         wait_interval=5,
+                         camera_list=None,
+                         exptimes_list=[],
```

Comment:
What happens if `exptimes_list` is empty? Do / should we have a predefined sequence of dark exposure times?

Reply:
Well, I reckon there should always be some targets in the config file, and the exposure times are taken from there to fill the list that is passed into the function `take_dark_fields`. So the only way for that list to be empty is that the targets config file is totally empty. If that happens, then `exptimes_list` would be empty. But I'm not sure if that's feasible since we always start the night with some targets in that file (and hence their exposure times), don't we? 
If there's a possibility that such a file be empty for any reason, then as you pointed out we should have a predefined list of exposure times. It totally depends on the config file for the targets.